src/commands/utils/ping.js

```diff
@@ -0,0 +1,9 @@
+const Command = require('../../classes/Command')
+
+module.exports = new Command(msg => {
+  const client = require('../../components/client')
+  const start = new Date(msg.timestamp)
+  msg.channel.createMessage('Pong!').then(c => {
+    c.edit(`Pong! \`${Math.floor(new Date(c.timestamp) - start)}ms, ${client.shards.random().latency}ms\``) // whatever, latency is pretty consistent across shards anyway
```

Comment:
Why `c` as a variable name when the return object is clearly a message? Wouldn't `m` make more sense?
```suggestion
  msg.channel.createMessage('Pong!').then(m => {
    m.edit(`Pong! \`${Math.floor(new Date(m.timestamp) - start)}ms, ${client.shards.random().latency}ms\``) // whatever, latency is pretty consistent across shards anyway
```